api/v1beta1/verticareplicator_types.go

```diff
@@ -43,6 +43,45 @@ type VerticaReplicatorSpec struct {
 	// be enabled on the source database. Custom username for source and target
 	// databases is not supported yet when TLS configuration is used.
 	TLSConfig string `json:"tlsConfig,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=async
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// Information of the target Vertica database to replicate to
+	Mode string `json:"mode"`
+}
+
+type VerticaReplicatorSourceDatabaseInfo struct {
```

Comment:
can you run `DEPLOY_WITH=olm make bundle`. It will update verticadb-operator.clusterserviceversion.yaml. Commit that file. I want to check if its content is still valid.